src/frontend/js/widgets/SyllabusCourseRunsList/components/CourseProductItem/index.spec.tsx

```diff
@@ -282,6 +282,94 @@ describe('CourseProductItem', () => {
       </Wrapper>,
     );
 
+    // Wait for product information to be fetched
+    await screen.findByRole('status', { name: 'Loading product information...' });
+    await screen.findByRole('heading', { level: 3, name: product.title });
+
+    const $price = screen.getByText(
```

Comment:
First time i see this notation `$` in our codebase, `const priceElement` should work.

Reply:
it's not the first time, you can go to the AdressForm test. I use the same syntax that is everywhere in this test file